libs/constants/src/lib/constants.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { constants } from './constants';
```

Comment:
I think this file can be removed completely. We don't need tests for constants :)